src/status_im/events.cljs

```diff
@@ -1,369 +1,372 @@
+            status-im.bootnodes.core
+            [status-im.bottom-sheet.core :as bottom-sheet]
+            [status-im.bottom-sheet.core2 :as bottom-sheet2]
+            status-im.browser.core
+            status-im.browser.permissions
+            [status-im.chat.models :as chat]
+            status-im.chat.models.images
+            status-im.chat.models.input
+            status-im.chat.models.loading
+            status-im.chat.models.transport
+            [status-im.constants :as constants]
+            status-im.contact.block
+            status-im.contact.chat
+            status-im.contact.core
+            status-im.currency.core
+            [status-im.ethereum.json-rpc :as json-rpc]
+            status-im.ethereum.subscriptions
+            status-im.fleet.core
+            status-im.http.core
+            [status-im.i18n.i18n :as i18n]
+            status-im.init.core
+            [status-im.keycard.core :as keycard]
+            status-im.log-level.core
+            status-im.mailserver.constants
+            [status-im.mailserver.core :as mailserver]
+            [status-im.multiaccounts.biometric.core :as biometric]
+            [status-im.multiaccounts.core :as multiaccounts]
+            status-im.multiaccounts.login.core
+            status-im.multiaccounts.logout.core
+            status-im.multiaccounts.update.core
+            [status-im.native-module.core :as status]
+            [status-im.navigation :as navigation]
+            status-im.notifications-center.core
+            status-im.activity-center.core
+            status-im.pairing.core
+            [status-im.popover.core :as popover]
+            status-im.profile.core
+            status-im.search.core
+            status-im.signals.core
+            status-im.stickers.core
+            status-im.transport.core
+            [status-im.ui.components.permissions :as permissions]
+            [status-im.ui.components.react :as react]
+            status-im.ui.screens.privacy-and-security-settings.events
+            [status-im.utils.dimensions :as dimensions]
+            [status-im.utils.fx :as fx]
+            status-im.utils.logging.core
+            [status-im.utils.universal-links.core :as universal-links]
+            [status-im.utils.utils :as utils]
+            [status-im.visibility-status-popover.core :as visibility-status-popover]
+            status-im.visibility-status-updates.core
+            status-im.waku.core
+            status-im.wallet.accounts.core
+            status-im.wallet.choose-recipient.core
+            [status-im.wallet.core :as wallet]
+            status-im.wallet.custom-tokens.core
+            [status-im.navigation.core :as navigation.core]
+            [status-im.navigation.state :as navigation.state]
+            [status-im.signing.core :as signing]
+            status-im.wallet-connect.core
+            status-im.wallet-connect-legacy.core
+            status-im.navigation2
+            status-im.navigation2.core))

```

Comment:
Can we add `[status-im.bottom-sheet.core2 :as bottom-sheet2]` to the end of the require list to not make changes in the whole list of requires ?